<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-L2504" ignored="0" name="GET Feeds - Net Income Summary" tests="18" failures="13" timestamp="02 Jun 2021 08:20:41 GMT" time="194.82" errors="0">
  <testcase name="@AfterMethod prepareAutomationReport" time="0.0" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yodlee.insights.yodleeApi.utils.InsightsHelper.printSummaryOfTests(InsightsHelper.java:1966)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.prepareAutomationReport(TestGetInsightsForIncomeExpenseOverview.java:314)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod prepareAutomationReport -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="6.636" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview"/>
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.986" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="10.396" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview"/>
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="10.272" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.743" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="8.756" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="9.936" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.212" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview"/>
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.234" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.195" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.028" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.41" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview"/>
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.04" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="8.425" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.396" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.337" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview"/>
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="7.93" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
  <testcase name="testGetNotificationsForIncomeExpenseOverview" time="9.277" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview">
    <failure type="java.lang.AssertionError" message="Test FAILED -&amp;gt; Expected  0 insights triggered But found 1 Insights for All Account ">
      <![CDATA[java.lang.AssertionError: Test FAILED -> Expected  0 insights triggered But found 1 Insights for All Account 
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForIncomeExpenseOverview.testGetNotificationsForIncomeExpenseOverview(TestGetInsightsForIncomeExpenseOverview.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForIncomeExpenseOverview -->
</testsuite> <!-- GET Feeds - Net Income Summary -->
